### DISCUSSION TITLE
Popup does not open in the native Android app

### DISCUSSION THREAD

**Discussion by panfil3** [markuspoerschke/iCal#254] [Q&A]
On android phones a file is generated and the download begins, and then when opening the downloaded file offers to record the event in the Google calendar. Is it possible to immediately open a popup for writing to the calendar, as it is implemented on the iPhone?

**Comment by markuspoerschke** [comment]
@panfil3 I moved your question into the Q&A section.

I do not have access to Android devices nor do I have experience developing apps for Android. Maybe someone of the community can help.
